src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/overview/GeneOverview.tsx

```diff
@@ -32,38 +34,82 @@ import JSONValue from 'src/shared/types/JSON';
 import { EntityViewerSidebarPayload } from 'src/content/app/entity-viewer/state/sidebar/entityViewerSidebarState';
 
 import styles from './GeneOverview.scss';
+import { EntityViewerParams } from 'src/content/app/entity-viewer/EntityViewer';
+import { Gene } from 'src/content/app/entity-viewer/types/gene';
+import { parseEnsObjectIdFromUrl } from 'src/shared/state/ens-object/ensObjectHelpers';
 
-type Props = {
+const QUERY = gql`
+  query Gene($genomeId: String!, $geneId: String!) {
+    gene(byId: { genome_id: $genomeId, stable_id: $geneId }) {
+      alternative_symbols
+      external_references {
+        accession_id
+        description
+        name
+        source {
+          name
+          id
+        }
+        url
+      }
```

Comment:
What do you want to use the external references for?

Reply:
I removed it